charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -29,9 +29,11 @@
   resources:
 {{ toYaml .Values.agents.containers.systemProbe.resources | indent 4 }}
   volumeMounts:
+    {{- if (not .Values.providers.gke.autopilot) }}
```

Comment:
IMO this change is not needed since system-probe is disabled in autopilot

Reply:
yes indeed